### ISSUE TITLE
🤗 Windows Defender - your main.exe file is a virus.

### ISSUE THREAD

**Issue by sdima75** [gofiber/fiber#1933]
Hello.
Forgive me for my bad English.
I had Debian 9 and NOD32 antivirus for a long time and everything worked fine.
Now I have Windows 10 and Windows Defender. I run - "go build" and Windows Defender tells me that my main.exe file is Trojan:Win32/Bearfoos.A!ml (https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3aWin32%2fBearfoos.A!ml)

Can you please tell me what is the problem here?
Is it a Windows Defender bug?

My code:
```package main

import "github.com/gofiber/fiber/v2"

func main() {
	// app := fiber.New()
	app := fiber.New()

	app.Get("/", func(c *fiber.Ctx) error {
		return c.SendString("Hello, World 👋!")
	})

	app.Listen(":3000")
}
```

This code compiles without any problems...
```
package main

import (
	"fmt"
	"net/http"
)

type msg string

func (m msg) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
	fmt.Fprint(resp, m)
}
func main() {
	msgHandler := msg("Hello from Web Server in Go")
	fmt.Println("Server is listening...")
	http.ListenAndServe("localhost:8181", msgHandler)
}
```

**Comment by ReneWerner87** [comment]
thank you for the report.

can someone with windows check if they can observe the same thing?

**Comment by KawaCode** [comment]
Try to update your Windows, in my windows there was not any problems with my .exe. Could be something else in your code, that your defender detects as virus.

**Comment by sdima75** [comment]
Hello . 
Forgive me for my poor English.
There was a Windows update on June 15.  It did not help.
Then the "virus" started to appear on my other projects where I use other packages.
On June 16th I still couldn't figure out what was causing it.
This morning I copied the project folder where I have "fiber" to another place. I built this package and there is no "virus". The same thing happened with my other projects. 

After lunch all my projects are built without "viruses"... ( in the old folders)...
I assume that it was a "bug" of Windows Defener... 

Forgive me for not figuring it out and using your time in such a useless way... I suggest to delete this thread, not to mislead people...
Good luck everyone!
